lib/engine/game/g_18_usa/entities.rb

```diff
@@ -295,9 +295,9 @@ module Entities
             name: 'P16 - Regional Headquarters',
             value: 60,
             revenue: 0,
-            desc: 'Regional Headquarters may be used to upgrade a green or brown non-metropolis city after phase 5 begins. ' \
-                  'May be placed on any existing normal city. Three of the track segments are optional and can be placed ' \
-                  'pointing toward water or other off map areas.',
+            desc: 'Regional Headquarters may be used to upgrade a green or brown non-metropolis city after phase 5 begins. '\
+                  'The X23 tile (RHQ label) may be placed on any existing normal city. Two of the track segments are optional '\
```

Comment:
The tile has 3 optional connections that are contiguous.

Recommend removing `(RHQ label)`. The tile number is enough and the description is already long.

Reply:
It's true that the private text says 3 of the segments are optional in the physical version, but there are no spaces on the board that would require more than 2 to be ignored. Should we still have it say 3?

I'll remove the (RHQ label) text.